tests/file_test.py

```
@@ -31,13 +31,15 @@ def get_expected_result(self):
         # pylint: disable=missing-function-docstring
         return self.expected_result
 
+
 class TestValidateUrl(unittest.TestCase):
     """
     _summary_
 
     Args:
         unittest (_type_): _description_
     """
```

Ruff (0.8.2)

Multi-line docstring summary should start at the first line

Remove whitespace after opening quotes

(D212)